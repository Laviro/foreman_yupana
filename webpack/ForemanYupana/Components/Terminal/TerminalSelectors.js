import { selectForemanYupana } from '../../ForemanYupanaSelectors';

export const selectTerminal = state => selectForemanYupana(state).terminal;
export const selectLogs = state => selectTerminal(state).logs;
