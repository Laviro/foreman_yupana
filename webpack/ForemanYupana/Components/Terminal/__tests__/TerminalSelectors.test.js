import { testSelectorsSnapshotWithFixtures } from 'react-redux-test-utils';
import { selectTerminal, selectLogs } from '../TerminalSelectors';

const state = {
  terminal: {
    logs: [],
  },
};

const fixtures = {
  'should return Terminal': () => selectTerminal(state),
  'should return Terminal Logs': () => selectLogs(state),
};

describe('Terminal selectors', () =>
  testSelectorsSnapshotWithFixtures(fixtures));
