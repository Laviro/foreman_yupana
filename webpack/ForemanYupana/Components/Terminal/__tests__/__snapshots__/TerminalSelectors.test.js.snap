// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Terminal selectors should return Terminal 1`] = `
Object {
  "logs": Array [],
}
`;

exports[`Terminal selectors should return Terminal Logs 1`] = `Array []`;
